packaging/windows/internal/cuda_install.bat

```diff
@@ -53,6 +54,23 @@ if not exist "%SRC_DIR%\temp_build\cudnn-10.0-windows10-x64-v7.4.1.5.zip" (
 
 goto cuda_common
 
+:cuda101
+
+if not exist "%SRC_DIR%\temp_build\cuda_10.1.243_426.00_win10.exe" (
+    curl -k -L http://developer.download.nvidia.com/compute/cuda/10.1/Prod/local_installers/cuda_10.1.243_426.00_win10.exe --output "%SRC_DIR%\temp_build\cuda_10.1.243_426.00_win10.exe"
```

Comment:
+1

Reply:
I uploaded it to https://ossci-windows.s3.amazonaws.com/cuda_10.1.243_426.00_win10.exe